ament_cmake_python/cmake/ament_python_install_module.cmake

```diff
@@ -55,7 +57,16 @@ function(_ament_cmake_python_install_module module_file)
     FILES "${module_file}"
     DESTINATION "${destination}"
   )
-  # TODO optionally compile Python file
+  if(NOT ARG_SKIP_COMPILE)
+    # compile Python files
+    install(CODE
+      "execute_process(
+        COMMAND
+        \"${PYTHON_EXECUTABLE}\" \"-m\" \"compileall\"
```

Comment:
It could, but until there is a use case when this is needed I wouldn't implement it.